app/src/index.ts

```diff
@@ -29,8 +29,10 @@ if (!process.env.fusionAuthURL) {
 const clientId = process.env.clientId;
 const clientSecret = process.env.clientSecret;
 const fusionAuthURL = process.env.fusionAuthURL;
+const fusionAuthRedirectURL = process.env.fusionAuthRedirectURL
 
 // Validate the token signature, make sure it wasn't expired
+#tag::validateUser[]
```

Comment:
Is a `#` a validate comment marker? I thought for JS you were supposed to use `//`?

Reply:
I should just take that out in any case, we're going through things in a much better way with your tagged sections.